dev/bots/test.dart

```diff
@@ -650,7 +677,7 @@ Future<void> _runFrameworkTests() async {
   Future<void> runMisc() async {
     print('${green}Running package tests$reset for directories other than packages/flutter');
     await _pubRunTest(path.join(flutterRoot, 'dev', 'bots'));
-    await _pubRunTest(path.join(flutterRoot, 'dev', 'devicelab'));
+    await _pubRunTest(path.join(flutterRoot, 'dev', 'devicelab'), forceSingleCore: true);
```

Comment:
@jmagman the issue was the unittests are run on multiple cores. In practice, we only run one devicelab test at a time so I opted to forceSingleCore here instead of resolving the race conditions. SG?

Reply:
LGTM